module-2/backend/Liam_barstad/week_three.txt

```diff
@@ -0,0 +1,71 @@
+## Week Three Recap
+
+### Instructions
+Fork this repository. Be sure to pull the latest changes to your local repo. Answer the questions to the best of your ability.
+
+Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
+
+Note: When you're done, submit a PR with a reflection in the comments about how this exercise went for you.
+
+### Questions
+
+1. What is the entry at the command line to create a new rails app?
+
+  rails new << project_name >> << options >>
+
+2. What do Models generally inherit from in rails?
+
+  ApplicationRecord
+
+3. What do Controllers generally inherit from in a rails project?
+
+  ApplicationController
+
+4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+
+  resources :horses, only: [:show]
+
+5. What rake task is useful when looking at routes, and what information does it give you?
+
+  Rake routes gives you the prefix, the HTTP verb, the url, and the ways to call the path in erb.
+
+6. What is an example of a route helper? When would you use them?
+
+
+
+7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+
+  '_url' will return the appendage to the url (e.g. "/jobs"), whereas the path will return the named path in rails (e.g. "jobs_path")
```

Comment:
_url will return the full path as if i was in the browser `localhost:3000/horses` while the _path is referencing how to reach this endpoint internally `/horses`